### PR TITLE
Update api-language.yml

### PR DESCRIPTION
**Pull request by bittlingmayer**

### CODE REVIEW

**Comment by cefoo** [comment]
What's the logic behind the key-value location? Are you grouping the codes that have comments at the beginning, and then it is alphabetic?

**Comment by bittlingmayer** [comment]
Yes, exactly, any funny business should be at the top.  The rest is very boring.

File: _data/api-language.yml [hunk 1]
```diff
@@ -17,6 +16,9 @@ microsoft:
   fil: tl # Macrolanguage, Microsoft for Tagalog
   mww: hmn # Macrolanguage, Microsoft for Hmong
   cz: cs # Czech, Omniscien
+  me: cnr # Montenegrin, Niutrans
+  mo: mn # Mongolian, Niutrans
+  jy: ka # Georgian, Niutrans
```
Comment (bittlingmayer) [inline _data/api-language.yml:21]: Arguably, these should go under `niutrans:`

It probably doesn't hurt to have them here, but it's hard to imagine another provider doing this exact same terrible deviation from the standard.

Comment (cefoo) [reply _data/api-language.yml:21]: I'll merge. We can reorganize it later if we decide to go with that option.

**Pull request merged**